production.Dockerfile

```diff
@@ -51,7 +51,8 @@ RUN yarn --frozen-lockfile
 # load entire codebase & build frontend
 COPY . /code/
 RUN yarn build \
-    && yarn --cwd plugins --frozen-lockfile --ignore-optional \
+    && yarn --cwd plugin-server --frozen-lockfile --ignore-optional \
+    && yarn build --cwd plugin-server \
```

Comment:
Why is this additional build necessary?

Reply:
So before we would pull the plugin server npm package, which comes readily built. Now, we have the plugin server code in here so need to build it ourselves.